src/include/buffer.h

```diff
@@ -671,9 +673,9 @@ namespace buffer CEPH_BUFFER_API {
   private:
     // my private bits
     buffers_t _buffers;
+    ptr* _carriage;
```

Comment:
might want to comment in the code to briefly document what `_carriage` is for.

Reply:
I will add one.